### PR TITLE
fix(fn): adjust slider handle position [ci visual]

### PR DESCRIPTION
**Pull request by N1XUS**
## Related Issue
none

## Description
Previously, slider handle was "tweaked" based on aria-valuenow property, which can be different in real-case scenario. Added wrapper which compensates handle offset so at the edge positions of the slider, handle would fit into the size of the slider itself.

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- n/a Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: src/fn/fn-slider.scss [hunk 1]
```diff
@@ -48,6 +49,14 @@ $slider-height: 0.5rem;
     opacity: $fn-disabled-opacity;
   }
 
+  &__wrapper {
+    @include fn-reset();
+
+    position: relative;
+    margin-left: $slider-wrapper-offset;
+    margin-right: $slider-wrapper-offset;
```
Comment (InnaAtanasova) [inline src/fn/fn-slider.scss:57]: You can use the `fn-set-margins-x` mixin here

File: src/fn/fn-slider.scss [hunk 2]
```diff
@@ -59,10 +68,10 @@ $slider-height: 0.5rem;
 
   &__track-range {
     @include fn-reset();
-    @include fn-set-position-left(0);
 
     z-index: 2;
-    width: 100%;
+    left: -$slider-wrapper-offset;
+    right: -$slider-wrapper-offset;
```
Comment (InnaAtanasova) [inline src/fn/fn-slider.scss:74]: for consistency, please use the mixins `fn-set-position-right` and `fn-set-position-left`. Thank you!

**Review by InnaAtanasova — APPROVED** [review]

File: src/fn/fn-slider.scss (hunk 2 above)
Comment (InnaAtanasova) [reply src/fn/fn-slider.scss:74]: Discussed with Denis, ignore my comment

Comment (N1XUS) [reply src/fn/fn-slider.scss:74]: No need since values are equal

File: src/fn/fn-slider.scss (hunk 1 above)
Comment (N1XUS) [reply src/fn/fn-slider.scss:57]: No need since values are equal

**Pull request merged**